Automattic/simplenote-ios
# Note Widget

### Fix
This PR is the next step in the widgets project.  Here I have added the ability to connect to the shared database with intents and widgets, parse those details, and then display them on the note widget.

<img height="400" src="https://cdn-std.droplr.net/files/acc_593859/cl7N41" />



### Test
1. Open simplenote while logged out
2. try to setup the note widget and confirm that you get an error
3. log into simplenote
4. return to the widget setup and tap on the Note option, you should see a list of all of your not deleted notes
5. choose a note with a title and some content in it
6. tap outside of the setup to return to the homescreen
7. after a second your note content should appear

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
> These changes do not require release notes.
